crates/gpui/src/platform/blade/shaders.wgsl

```diff
@@ -266,7 +273,7 @@ fn fs_quad(input: QuadVarying) -> @location(0) vec4<f32> {
                     saturate(0.5 - inset_distance));
     }
 
-    return color * vec4<f32>(1.0, 1.0, 1.0, saturate(0.5 - distance));
+    return color * alpha_factor(saturate(0.5 - distance));
```

Comment:
`alpha_factor` should probably also account for the alpha value from `color`

Reply:
good point! refactoring now